apps/base-docs/tutorials/docs/1_thirdweb-unreal-nft-items.md

```diff
@@ -0,0 +1,773 @@
+---
+title: 'Thirdweb and Unreal - NFT Items'
+slug: /thirdweb-unreal-nft-items
+description: Learn how to use NFTs as in-game items using Thirdweb and Unreal.
+author: briandoyle81
+keywords: [Solidity, ERC-721, token, NFT, thirdweb, Unreal, c++, blueprints, onchain games]
+tags: ['nft']
+difficulty: hard
+hide_table_of_contents: false
+displayed_sidebar: null
+---
+
+[thirdweb] provides a number of contracts and tools for building onchain. Their [Gaming SDK] enables seamless onboarding, cross-platform support, and provides many other features. It's compatible with [Unreal Engine] and can be used to enable onchain elements in your games.
+
+In this tutorial, you'll learn how to add NFT item usage on top of the demo game you build in their [Unreal Engine Quickstart]. Specifically, you'll use an NFT collection of random colors to change the color of the player's race car.
+
+![Color changing car nft](../../assets/images/build-with-thirdweb/car-color-nft.gif)
+
+---
+
+## Objectives
+
+By the end of this tutorial you should be able to:
+
+- Pull a user's NFTs into an Unreal Engine Game
+- Apply elements from the NFT to game entities
+- Award NFTs to players for game accomplishments or actions
+
+---
+
+## Prerequisites
```

Comment:
Maybe add HDD requirements + XCode?

I deleted Xcode to free up space and realized I needed xcode to run Unreal Engine.

Reply:
Add Postgres to prereqs

Reply:
Xcode is covered under the Unreal Engine one, they have you install it in the setup and we're assuming that is already done.

For postgres, you just need to install it, which is inline below.  You don't need to know anything about it or do anything with it for this tutorial.